### PR TITLE
Fix abstract-integration-testing package

### PR DESCRIPTION
**Pull request by CyberHoward**
Updates the abstract-intergation-testing package with the v2 changes

### CODE REVIEW

File: framework/contracts/native/account-factory/Cargo.toml
Comment (Buckram123) [inline framework/contracts/native/account-factory/Cargo.toml:47]: This branch is outdated, this file does not exist in `develop/v2`

**Review by Buckram123 — APPROVED** [review]
LGTM after resolving merge conflicts

**Comment by CyberHoward** [comment]
Removing account factory broke this branch but is fixed in #455 so closing this in favor of that branch.

**Pull request closed without merging**
